pkg/ccl/testccl/sqlccl/sqlccl_test: TestStandbyRead failed

pkg/ccl/testccl/sqlccl/sqlccl_test.TestStandbyRead [failed](https://tanzanite.cluster.engflow.com/invocations/default/693db6c9-3ba0-432c-abe1-c5fbd4dfee86?testReportRun=1&testReportShard=14&testReportAttempt=1#targets-Ly9wa2cvY2NsL3Rlc3RjY2wvc3FsY2NsOnNxbGNjbF90ZXN0) on master @ [ee6710a81306e851d85a6f664eebfa249a17d16c](https://github.com/cockroachdb/cockroach/commits/ee6710a81306e851d85a6f664eebfa249a17d16c):

Fatal error:

```
panic: test timed out after 41m37s
running tests:
	TestStandbyRead (41m37s)

```
Stack: 

```
goroutine 456000 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x265
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x45
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestStandbyRead
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestStandbyRead445092734
    test_log_scope.go:76: use -show-logs to present logs inline
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=14</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStandbyRead.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-43517

pkg/ccl/testccl/sqlccl/sqlccl_test.TestStandbyRead [failed](https://tanzanite.cluster.engflow.com/invocations/default/8bc3b4f8-00ce-4012-8489-c39022b4e61b?testReportRun=1&testReportShard=14&testReportAttempt=1#targets-Ly9wa2cvY2NsL3Rlc3RjY2wvc3FsY2NsOnNxbGNjbF90ZXN0) on master @ [52e38aafd852b646456000a04e1dc440627d2c6d](https://github.com/cockroachdb/cockroach/commits/52e38aafd852b646456000a04e1dc440627d2c6d):

Fatal error:

```
panic: test timed out after 41m37s
running tests:
	TestStandbyRead (41m37s)

```
Stack: 

```
goroutine 166774 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x265
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x45
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestStandbyRead
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestStandbyRead2603529192
    test_log_scope.go:76: use -show-logs to present logs inline
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=14</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStandbyRead.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestStandbyRead [failed](https://tanzanite.cluster.engflow.com/invocations/default/79a3947e-b05f-4d7b-bb3b-28f6568fa2d1?testReportRun=1&testReportShard=14&testReportAttempt=1#targets-Ly9wa2cvY2NsL3Rlc3RjY2wvc3FsY2NsOnNxbGNjbF90ZXN0) on master @ [82b1fda15c4616713b278c447d24b0ab5416e511](https://github.com/cockroachdb/cockroach/commits/82b1fda15c4616713b278c447d24b0ab5416e511):

Fatal error:

```
panic: test timed out after 41m37s
running tests:
	TestStandbyRead (41m37s)

```
Stack: 

```
goroutine 210792 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x265
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x45
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestStandbyRead
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestStandbyRead217122439
    test_log_scope.go:76: use -show-logs to present logs inline
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=14</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStandbyRead.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestStandbyRead [failed](https://tanzanite.cluster.engflow.com/invocations/default/c9db6290-4cd0-4b62-9618-a21b407222ac?testReportRun=3&testReportShard=14&testReportAttempt=1#targets-Ly9wa2cvY2NsL3Rlc3RjY2wvc3FsY2NsOnNxbGNjbF90ZXN0) on master @ [2afb5fa158338113de3f7ec9ba48a33263440061](https://github.com/cockroachdb/cockroach/commits/2afb5fa158338113de3f7ec9ba48a33263440061):

Fatal error:

```
panic: test timed out after 41m37s
running tests:
	TestStandbyRead (41m37s)

```
Stack: 

```
goroutine 435032 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x265
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x45
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestStandbyRead
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestStandbyRead3637730878
    test_log_scope.go:76: use -show-logs to present logs inline
I241026 11:05:48.571833 1 (gostd) rand.go:245  [-] 1  random seed: -7496877116538071951
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=3</code>
 - <code>shard=14</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStandbyRead.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestStandbyRead [failed](https://tanzanite.cluster.engflow.com/invocations/default/de8d3931-92e2-4982-902f-0ae7d4b8c4d6?testReportRun=1&testReportShard=14&testReportAttempt=1#targets-Ly9wa2cvY2NsL3Rlc3RjY2wvc3FsY2NsOnNxbGNjbF90ZXN0) on master @ [1b5c41939197efb0fa50ded9795e2e83f5c1dd34](https://github.com/cockroachdb/cockroach/commits/1b5c41939197efb0fa50ded9795e2e83f5c1dd34):

Fatal error:

```
panic: test timed out after 41m37s
running tests:
	TestStandbyRead (41m37s)

```
Stack: 

```
goroutine 459222 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x265
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x45
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestStandbyRead
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestStandbyRead3549280388
    test_log_scope.go:76: use -show-logs to present logs inline
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=14</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStandbyRead.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/testccl/sqlccl/sqlccl_test.TestStandbyRead [failed](https://tanzanite.cluster.engflow.com/invocations/default/69b54da0-7485-45ee-aac9-a5458f57d49a?testReportRun=1&testReportShard=14&testReportAttempt=1#targets-Ly9wa2cvY2NsL3Rlc3RjY2wvc3FsY2NsOnNxbGNjbF90ZXN0) on master @ [769a47f67b95f31ad13208764226a67a0baa200b](https://github.com/cockroachdb/cockroach/commits/769a47f67b95f31ad13208764226a67a0baa200b):

Fatal error:

```
panic: test timed out after 41m37s
running tests:
	TestStandbyRead (41m37s)

```
Stack: 

```
goroutine 523275 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x265
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x45
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestStandbyRead
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestStandbyRead453241214
    test_log_scope.go:76: use -show-logs to present logs inline
I241028 10:58:45.102418 1 (gostd) rand.go:245  [-] 1  random seed: 1076816612368532698
```
</p>
</details>

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=14</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestStandbyRead.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/ccl/testccl/sqlccl/standby_read_test.go
+++ b/pkg/ccl/testccl/sqlccl/standby_read_test.go
@@ -15,6 +15,7 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/sql/catalog/replication"
 	"github.com/cockroachdb/cockroach/pkg/testutils"
 	"github.com/cockroachdb/cockroach/pkg/testutils/serverutils"
+	"github.com/cockroachdb/cockroach/pkg/testutils/skip"
 	"github.com/cockroachdb/cockroach/pkg/testutils/sqlutils"
 	"github.com/cockroachdb/cockroach/pkg/util/leaktest"
 	"github.com/cockroachdb/cockroach/pkg/util/log"
@@ -25,6 +26,8 @@ import (
 func TestStandbyRead(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
+
+	skip.UnderDuress(t, "slow test")
 
 	testcases := []struct {
 		standby  bool

